.gitattributes

```diff
@@ -0,0 +1,3 @@
+.* export-ignore
+*.md export-ignore
```

Comment:
I would keep markdowns in distros, yet `phpunit.xml.dist` looks line one to add to ignores.

Reply:
it is not, still markdowns are excluded and `phpunit.xml.dist` isn't.

Reply:
Strange, will have a look later